src/app/components/media/ReportDesigner/ReportDesignerForm.js

```diff
@@ -90,292 +55,490 @@ const ReportDesignerForm = (props) => {
+                      }
+                      disabled={props.pending || props.disabled}
+                      maxHeight="48px"
+                      autoGrow="false"
+                      placeholder={placeholder}
+                      onBlur={(e) => {
+                        const newValue = e.target.value;
+                        let newUrl = newValue;
+                        if (newValue.trim().length !== 0 && !/^https?:\/\//.test(newUrl)) {
+                          newUrl = `https://${newUrl}`;
+                        }
+                        props.onUpdate('published_article_url', newUrl);
+                      }}
+                    />
+                  )}
+                </FormattedMessage>
+              </div>
+            </> : null
+          }
+          { data.use_visual_card && !data.use_text_message ?
+            <>
+              <ToggleButtonGroup
+                key={`dark-overlay-${data.language}`}
+                label={<FormattedMessage id="reportDesigner.theme" defaultMessage="Theme" description="Label used for choice on theme of report light versus dark" />}
+                className={inputStyles['form-fieldset-field']}
+                variant="contained"
+                onChange={(e, newValue) => {
+                  props.onUpdate('dark_overlay', Boolean(newValue));
+                }}
+                exclusive
+              >
+                <ToggleButton
+                  selected={Boolean(!data.dark_overlay)}
+                  value={Boolean(false)}
+                  key="false"
+                  className="int-report__button--report-type"
+                >
+                  <FormattedMessage id="reportDesigner.lightTheme" defaultMessage="Light" description="Label used for button toggling on the light themed report" />
+                </ToggleButton>
+                <ToggleButton
+                  selected={Boolean(data.dark_overlay)}
+                  value={Boolean(true)}
+                  key="true"
+                  className="int-report__button--report-type"
+                >
+                  <FormattedMessage id="reportDesigner.darkTheme" defaultMessage="Dark" description="Label used for button toggling on the light themed report" />
+                </ToggleButton>
+              </ToggleButtonGroup>
+              <div className={styles['report-rating-wrapper']}>
+                <div className={styles['report-rating-headline']}>
+                  <ColorPicker
+                    color={data.theme_color}
+                    onChange={handleChangeColor}
+                  />
+                  <FormattedMessage
+                    id="reportDesigner.textStatusPlaceholder"
+                    defaultMessage="Add a status to this report"
+                    description="Placeholder for report status field"
+                  >
+                    { placeholder => (
+                      <LimitedTextArea
+                        required={Boolean(false)}
```

Comment:
Not really the scope of this PR, but I just wanted to vent that required as default is annoying :smiling_face_with_tear:

Reply:
yeah...when I ran into this I thought the same thing. maybe next time i do a pure component improvement pass to match the design system I will tick this off